airbrakes/hardware/imu.py

```diff
@@ -123,69 +184,27 @@ def _query_imu_for_data_packets(self, port: str) -> None:
             self._fetch_data_loop(port)
 
     def _fetch_data_loop(self, port: str) -> None:
```

Comment:
also deleted - move port to self variable?

Reply:
no

Reply:
yeah that's why I was putting those comment separators, even in the init. Since we have long running processes, and many methods, an object oriented approach is still the best imo.